heft-plugins/heft-webpack5-plugin/package.json

```diff
@@ -14,16 +14,18 @@
   "scripts": {
     "build": "heft build --clean",
     "start": "heft test --clean --watch",
-    "_phase:build": "heft build --clean",
-    "_phase:test": "heft test --no-build"
+    "_phase:build": "heft run --only build -- --clean",
+    "_phase:test": "heft run --only test -- --clean"
   },
   "peerDependencies": {
     "@rushstack/heft": "^0.45.14",
     "webpack": "~5.68.0"
   },
   "dependencies": {
     "@rushstack/node-core-library": "workspace:*",
-    "webpack-dev-server": "~4.7.4"
+    "@types/tapable": "1.0.6",
+    "tapable": "1.1.3",
+    "webpack-dev-server": "~4.9.3"
```

Comment:
Why wasn't this a peerDependency before?

Reply:
I'm unsure since I haven't really worked on the webpack plugin, but my guess is that bringing a version along with us is safer. @dmichon-msft had to make changes when upgrading from 4.7.x to 4.9.x, so I'm guessing that it's just easier to bring our own copy.